File: StudentManagement/Project/StudentManagementSystem_Aitrich/Domain/Models/TrialStudent.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Domain.Enums;
using Microsoft.EntityFrameworkCore;

namespace Domain.Models
{
    public class TrialStudent
    {
        [Key]
        
        public Guid TrialStudentId { get; set; }

        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Address { get; set; }
        public string Email { get; set; }
		public StudentStatus studentStatus { get; set; }
		public string Phone { get; set; }
        public DateTime RegistrationTime { get; set; }
        [ForeignKey(nameof(Course))]
       public Guid? CourseId { get; set; }
        public virtual Course Courses { get; set; }
        public virtual RegistrationFee? RegistrationFee { get; set; }



    }
}

File: StudentManagement/Project/StudentManagement_Angular/StudentManagementAdmin/src/app/trial-student/Models/trialStudent.ts
export interface trialStudent{
    trialStudentId: string;
    firstName: string;
    lastName: string;
    address: string;
    email: string;
    phone: string;
    status: StudentStatus;
}
export enum StudentStatus {
  Registered = 'Registered',
  Enrolled = 'Enrolled',
  Deleted = 'Deleted'
}